src/lib/hooks/useConnect/setupConnection.ts

```diff
@@ -37,6 +40,7 @@ export function setUpParams({
       new OpenChannelModule(),
     ],
     newInstance: true,
+    ...sdkInitParams,
```

Comment:
undefined/null check necessary?

Reply:
Trying spread from undefined happens nothing actually but always better to be explicit :) a2f0541863b6f4ef5460af4a3792d893890f16cf

Reply:
hmmm have crashed on me once before(dont remember the env)